models/model.js

```diff
@@ -5,3 +5,11 @@ exports.fetchCategories = () => {
     return result.rows;
   });
 };
+
+exports.fetchReviews = () => {
+  return db
+    .query("SELECT * FROM reviews ORDER BY created_at DESC;")
```

Comment:
In order to add comment_count, you need a JOIN query that counts the number of comments each review has. COUNT will return a string value, so think about how this will interact with your existing test.